src/main/java/software/bernie/techarium/client/tile/render/BotariumRenderer.java

```diff
@@ -20,9 +20,10 @@
 import net.minecraft.util.math.vector.Quaternion;
 import net.minecraftforge.client.model.data.EmptyModelData;
 import net.minecraftforge.common.Tags;
+import software.bernie.example.client.model.tile.BotariumModel;
 import software.bernie.geckolib3.renderers.geo.GeoBlockRenderer;
 import software.bernie.techarium.client.RenderUtils;
-import software.bernie.techarium.client.tile.model.BotariumModel;
+import software.bernie.techarium.client.tile.model.ArboretumModel;
```

Comment:
Wrong model?

Reply:
Its not used at all, just an artifact from testing